### PR TITLE
SRT Receiver fixes

### PR DESCRIPTION
**Pull request by kierank**

### CODE REVIEW

**Review by funman — COMMENTED** [review]
acks should not be used beyond this point.

Did you see a crash or is it preventive?

File: lib/upipe-srt/upipe_srt_receiver.c [hunk 1]
```diff
@@ -1318,6 +1318,7 @@ static void upipe_srt_receiver_free(struct upipe *upipe)
     upipe_throw_dead(upipe);
 
     free(upipe_srt_receiver->acks);
+    upipe_srt_receiver->acks = NULL;
```
Comment (funman) [inline lib/upipe-srt/upipe_srt_receiver.c:1334]: acks should not be used beyond this point.

Did you see a crash or is it preventive?

Comment (kierank) [reply lib/upipe-srt/upipe_srt_receiver.c:1334]: I had a segfault on this, yes. And have not seen it since addding NULL.

Comment (funman) [reply lib/upipe-srt/upipe_srt_receiver.c:1334]: That's weird, I hope we are not hiding another serious bug

**Pull request merged**